app/models/manageiq/providers/redhat/infra_manager/provision/configuration/network.rb

```diff
@@ -80,4 +91,37 @@ def configure_vnic(network_hash, name, vnic)
       :logger    => _log
     )
   end
+
+  def configure_v4_vnics(requested_vnics)
+    destination.ext_management_system.with_provider_connection(:version => 4) do |connection|
+      nics_service = connection.system_service.vms_service.vm_service(destination.uid_ems).nics_service
```

Comment:
I would probably pack it as vm.provider_nics_service to ManageIQ::Providers::Redhat::InfraManager::Vm, similar to provider_object method. And I wonder if it can leverage the provider_object method?